tests/test_api.py

```diff
@@ -9,7 +9,7 @@ async def test_connect_client():
     with patch("moonraker_api.MoonrakerClient"), patch(
         "moonraker_api.websockets.websocketclient.WebsocketClient.connect"
     ), patch("moonraker_api.websockets.websocketclient.WebsocketClient.disconnect"):
-        moonraker_api = MoonrakerApiClient("notaURL", None)
+        moonraker_api = MoonrakerApiClient("notaURL", None, port="1234")
```

Comment:
we probably want to test for a bit more.

- Good port
- Bad port
- No port
- Not an int

Reply:
I added the check in test_config_flow, i'm not sure to understand what test_api is actually testing...